spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
+=== MicroProfile 5.0 Release Candidate 1 (Q32021)
+// MicroProfile 5.0-RC1 is the 16th platform release for the
+// https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project].
+MicroProfile 5.0-RC1 is based on Jakarta EE 8, the first MicroProfile release to
+be based on Jakarta EE (replacing the role of Java EE).
+Although Jakarta EE 8 is API backwards compatible with Java EE 8, Jakarta EE replaces Java EE dependencies with
+Jakarta EE dependencies for all MicroProfile specifications.
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC4/microprofile-config-spec-3.0-RC4.html#_incompatible_changes[Config]
+* https://download.eclipse.org/microprofile/staging/microprofile-fault-tolerance-4.0-RC2/microprofile-fault-tolerance-spec-4.0-RC2.html#_incompatible_changes[Fault Tolerance]
+* https://github.com/eclipse/microprofile-health/releases/tag/4.0-RC2[Health TODO]
+* https://github.com/eclipse/microprofile-metrics/releases/tag/4.0-RC1[Metrics TODO]
+* https://download.eclipse.org/microprofile/staging/microprofile-open-api-3.0-RC1/microprofile-openapi-spec-3.0-RC1.html#_incompatible_changes[Open API]
+* https://download.eclipse.org/microprofile/microprofile-rest-client-3.0-RC2/microprofile-rest-client-spec-3.0-RC2.html[Rest Client - BROKEN LINK]
+* https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.0-RC2[JWT TODO]
+* https://download.eclipse.org/microprofile/staging/microprofile-opentracing-3.0-RC2/microprofile-opentracing-spec-3.0-RC2.html#_incompatible_changes[Open Tracing]
+
+If you are still dependent on Java EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/4.1[4.1 release of MicroProfile].
+If you are still dependent on Java EE 7, please consider using the https://github.com/eclipse/microprofile/releases/tag/1.4[1.4 release of MicroProfile].
+
+The complete list of MicroProfile 5.0-RC1 specifications includes:
+
```

Comment:
The versions between brackets need to match with the links

Reply:
It matches with the link, does not it? are you referring to the line [56] recommending MP1.4 if you still using Java EE 7?

Reply:
I think these above are fine, but these below are not aligned yet. E.g.: /tag/3.0-RC4[MicroProfile Config 2.0]

Reply:
Ohhhhh I see what you meant, these are fixed now!!! thanks